.github/workflows/run-all-tests.yaml

```diff
@@ -75,11 +75,30 @@ jobs:
         run: npm run build --workspace=@jazzer.js/fuzzer
       - name: run all fuzz tests
         run: node fuzztests/runFuzzTests.js
+  end-to-end:
+    name: end-to-end
+    runs-on: ubuntu-20.04
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: "npm"
+          registry-url: https://registry.npmjs.org
```

Comment:
Is specifying the registry necessary here?

Reply:
No idea, I just copied it from the other tests in this file.